kernel/src/engine/arrow_conversion.rs

```diff
@@ -107,10 +107,10 @@ impl TryFrom<&DataType> for ArrowDataType {
                         // timezone. Stored as 4 bytes integer representing days since 1970-01-01
                         Ok(ArrowDataType::Date32)
                     }
-                    PrimitiveType::Timestamp => {
-                        // Issue: https://github.com/delta-io/delta/issues/643
```

Comment:
The issue still seems to be open/applicable?
(I don't know what the immediate answer is, tho, given that there seem to be at least three ways to physically represent a timestamp?)